_tuning-your-cluster/availability-and-recovery/segment-replication/index.md

```diff
@@ -12,29 +12,29 @@ redirect_from:
 
 # Segment replication
 
-With segment replication, segment files are copied across shards instead of documents being indexed on each shard copy. This improves indexing throughput and lowers resource utilization at the expense of increased network utilization.
+Segment replication involves copying segment files across shards instead of indexing documents on each shard copy. This approach enhances indexing throughput and reduces resource utilization, but increases network utilization. Segment replication is the first feature in a series of features designed to decouple reads and writes in order to lower compute costs.
 
 When the primary shard sends a checkpoint to replica shards on a refresh, a new segment replication event is triggered on replica shards. This happens:
 
 - When a new replica shard is added to a cluster.
 - When there are segment file changes on a primary shard refresh.
 - During peer recovery, such as replica shard recovery and shard relocation (explicit allocation using the `move` allocation command or automatic shard rebalancing).
 
-Segment replication is the first feature in a series of features designed to decouple reads and writes in order to lower compute costs.
-
 ## Use cases
 
+Segment replication can be applied in a variety of scenarios, including:
+
 - Users who have high write loads but do not have high search requirements and are comfortable with longer refresh times.
```

Comment:
"Users" are not "scenarios", so the first two items do not complete the stem. The stem is fine, but I would rephrase the first two items.

Reply:
Revised.